app/furniture/marketplace/tag.rb

```diff
@@ -10,5 +10,13 @@ class Tag < Record
 
     attr_accessor :marketplace
     location(parent: :marketplace)
+
+    positioned
+
```

Comment:
The only merge-blocking issue I see in this PR right now is sorting out this `positioned` scope.